src/main/java/nl/tudelft/ewi/devhub/server/database/entities/GroupMembership.java

```diff
@@ -5,20 +5,29 @@
 import javax.persistence.Entity;
 import javax.persistence.Id;
 import javax.persistence.JoinColumn;
+import javax.persistence.JoinColumns;
 import javax.persistence.ManyToOne;
 import javax.persistence.Table;
+import javax.persistence.UniqueConstraint;
 
 import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
 
 @Data
 @Entity
 @SuppressWarnings("serial")
 @Table(name = "group_memberships")
+@ToString(of={"group", "user"})
+@EqualsAndHashCode(of={"group", "user"})
 public class GroupMembership implements Serializable {
 
 	@Id
 	@ManyToOne
-	@JoinColumn(name = "group_id")
+	@JoinColumns({
+		@JoinColumn(name = "course_id"),
+		@JoinColumn(name = "group_number")
+	})
 	private Group group;
```

Comment:
`@Id` on `@ManyToOne` columns is not allowed. We should use [`@MapsId`](http://docs.oracle.com/javaee/6/api/javax/persistence/MapsId.html) instead.